modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlSynchronousApiTest.java

```diff
@@ -226,6 +227,7 @@ public void select() throws ExecutionException, InterruptedException {
     }
 
     @Test
+    @Disabled ("IGNITE-17806 Rollback SQL automatically created transaction when an error has happened")
```

Comment:
Do you mean that "automatically created" is implicit? If true how it's possible to rollback implicit transactions?

Reply:
I mean implicit for a user, but explicit for SQL engine.